baseapp-chats/baseapp_chats/graphql/mutations.py

```diff
@@ -320,9 +335,35 @@ def mutate_and_get_payload(
                 oldest_remaining_participant.role = ChatRoomParticipantRoles.ADMIN
                 oldest_remaining_participant.save(update_fields=["role"])
 
-        # Update room
-        # TODO: Remeber to include added participants into count when implementing that feature
-        room.participants_count = room.participants_count - len(removed_participants)
+        # Adding new participants
+        unique_participants_pks = list(set(add_participants_pks))
+        existing_participants_pks = ChatRoomParticipant.objects.filter(
+            Q(room=room) & Q(profile__pk__in=unique_participants_pks)
+        ).values_list("profile__pk", flat=True)
+
+        new_participants = [
+            participant
+            for participant in unique_participants_pks
+            if int(participant) not in existing_participants_pks
+        ]
+
+        new_participants_profiles = Profile.objects.filter(pk__in=new_participants)
+
+        created_participants = ChatRoomParticipant.objects.bulk_create(
+            [
+                ChatRoomParticipant(
+                    profile=participant,
+                    room=room,
+                    role=ChatRoomParticipantRoles.MEMBER,
+                    accepted_at=timezone.now(),
+                )
+                for participant in new_participants_profiles
```

Comment:
you could probably just do `for participant_id in new_participants`, then `profile_id=participant_id` on line 355, and then get rid of the query on line 350.